tests/unit/post_processors/test_otel_metrics_results_processor.py

```
@@ -0,0 +1,619 @@
+
+        def export(
+            self, metrics_data: object, timeout_millis: float = 10000, **kwargs: object
+        ) -> FakeMetricExportResult:
+            self.export_calls.append((metrics_data, timeout_millis, kwargs))
+            return FakeMetricExportResult.SUCCESS
+
+        def force_flush(self, timeout_millis: int = 30000) -> bool:
+            self.force_flush_calls.append(timeout_millis)
+            return True
+
+        def shutdown(self, timeout_millis: float = 30000, **kwargs: object) -> None:
```

Ruff (0.15.1)

[warning] Unused method argument: `timeout_millis`

(ARG002)